text/0021-versioning-releases.md

```diff
@@ -0,0 +1,45 @@
+# Releasing and Versioning of Openmrs SingleSpa Apps
+- Start Date: 2020/01/21
+- RFC PR: https://github.com/openmrs/openmrs-rfc-frontend/pull/1
+
+## Definition
+OpenMRS: refers to the openmrs community.
+
+Micro-Frontends Squad: refers to the OpenMRS Community members responsible for creating and maintaining OpenMRS SPAs, under the OpenMRS Micro-Frontends Project.
+
+OpenMRS SPA: refers to an OpenMRS Micro-Frontend app. An example is openmrs-esm-patient-chart. See https://github.com/openmrs/openmrs-esm-patient-chart
+
+CDN: Content Distribution Network such as jsdelivr. See https://www.jsdelivr.com/
+
+MAJOR Version: version when you make incompatible API changes
+
+MINOR Version: version when you add functionality in a backwards compatible manner
+
+PATCH Version: version when you make backwards compatible bug fixes
+
+## Decision
+OpenMRS SPA applications should have releases that are versioned so that each implementor/distribution can decide when to upgrade to a certain version, containing features that the implementor/distribution needs. 
+
+The releases should have a maintenance branch for bug fixes and patches so that implementors running a particular version of an app can benefit from recent bug fixes. 
+
+I propose that we have maintenance release branches for both Major and Minor releases, as the motivation for this is to allow implementors/distributions control on when to start using a certain feature, that could be in a certain minor or major release. 
+
+At this time, OpenMRS MF Squad will not maintain older released versions in terms of porting back features and bug fixes to those releases. An implementor/distribution who wishes to benefit from bug fixes, and is on an earlier major or minor release will have to back-port those bug fixes to the older release themselves. Such an individual will need write access to the Github repo hosting the SPA application of interest. 
+
+When to do a major or minor release will be request-driven. An implementor/distribution who may wish to benefit from features not yet released will need to request the Micro-Frontends Squad for a certain feature to be released, in a minor or major version as seen fit.  
```

Comment:
I think we should feel free to version and release as often as our progress warrants. If we're getting a lot of requests to "please release X," that will mean we are not releasing often enough. I would just get rid of this line, since what I'm suggesting is what we already do.